File: ASP.NETCoreWebApplicationWithAngular/Models/OrdersBooks.cs
namespace ASP.NETCoreWebApplicationWithAngular.Models
{
    public class OrdersBooks
    {
        public int OrdersBooksID { get; set; }
        public int OrderID { get; set; }
        public Order Order { get; set; }
        public int BookId { get; set; }
        public Book Book { get; set; }
    }
}

File: ASP.NETCoreWebApplicationWithAngular/ClientApp/src/app/shared/OrdersBooks.ts
import {User} from "./User";
import {Book} from "./Book";
import {Order} from "./Order";

export class OrdersBooks {
  constructor(
    public ordersBooksID?: number,
    public orderID?: number,
    public order?: Order,
    public bookId?: number,
    public book?: Book
  ) {
  }
  }
